src/App.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
-
-function App() {
-  return (
-    <h1>
-      Hello Wors
-    </h1>
-  );
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import Form from './Components/Form/Form';
+import WeatherDetails from './Components/WeatherDetails/WeatherDetails';
+
+class App extends Component {
+  state = {
+    weatherData: '',
+  };
+
+  handleState = data => {
```

Comment:
change name to `setWehaterData`  or something like this. 
Remove unnecessary braces 

```
setWheaterData = data =>  this.setState({
      weatherData: data
});
```